pyblish_bumpybox/plugins/validate_scene_version.py

```diff
@@ -6,8 +8,12 @@ class ValidateSceneVersion(pyblish.api.ContextPlugin):
 
     order = pyblish.api.ValidatorOrder
     label = "Scene Version"
+    optional = True
 
     def process(self, context):
 
-        msg = "Could not find a version number in the scene name."
-        assert context.has_data("version"), msg
+        name, ext = os.path.splitext(context.data("currentFile"))
+        msg = "Could not find a version number in the scene name. Please add "
+        msg += "v[number] to the scene name, for example: "
+        msg += "\"{0}_v001{1}\".".format(name, ext)
+        assert "version" in context.data, msg
```

Comment:
> Now that will be a question of the style we want to stick to. I've been using double quotes throughout for consistency, and it aligns with pyblish's coding style. I would prefer to stick with one style throughout.

You brought up the question on coding style around quotes. :)

Either approach is valid and pyblish seems to use them interchangeably anyway. https://github.com/pyblish/pyblish-base/blob/master/tests/test_context.py#L47  https://github.com/pyblish/pyblish-base/blob/master/tests/test_context.py#L108

I don't really mind which you use, just that really you shouldn't need to escape stuff within a string. That's the whole reason they're interchangeable as I understand it.